Sprint-3/revise/implement/is-prime.js

```diff
@@ -0,0 +1,9 @@
+function isPrime(num) {
+  if (num <=1) return false;
+  for(let i = 2; i < num; i++) {
```

Comment:
You can possibly improve the performance of the code in the following manners:
- Return `true` if `num` is 2
- Return `false` if `num` is an even number (we have already checked 2)

- Use a loop to check if `num` can be fully divided by an odd number >= 3 (but <= `Math.sqrt(num)`)
- In the loop, avoid calling `Math.sqrt(num)` repeatedly by first assigning the value of `Math.sqrt(num)` to a variable once, and then refer to the variable in the condition of the loop.

Reply:
Alright, let me try implement the suggestions.